src/barkhaPJun20/PalindromeString.java

```diff
@@ -0,0 +1,22 @@
+package barkhaPJun20;
+
+public class PalindromeString {
+	
+	void palindromeString (String originalName) {
+		String name1=originalName;
```

Comment:
You have not changed name1 value anywhere in the program. no need to assign originalName to name1. if you are changing originalName value then you have to store to it.